openedx/features/student_certificates/tests/test_helpers.py

```diff
@@ -127,3 +129,12 @@ def test_get_course_display_name_by_uuid(self):
         verify_uuid = 'test_uuid'
         with self.assertRaises(Http404):
             get_course_display_name_by_uuid(verify_uuid)
+
+    def test_get_certificate_pdf_name(self):
+        """
+        Test certificate PDF name
+        """
+        course_display_name = CourseOverview.objects.get(id=self.course.id).display_name.replace(' ', '')
+        expected_pdf_name = 'PhilanthropyUniversity_{display_name}'.format(display_name=course_display_name)
+        certificate_pdf_name = get_certificate_pdf_name(self.certificate.verify_uuid)
```

Comment:
Maybe it'll be better if we stick to _expected/actual_ convention. `certificate_pdf_name` can be renamed to `actual_pdf_name`.